test/specs/from.coffee

```diff
@@ -0,0 +1,30 @@
+Observable = require('zen-observable')
+Rx = require('@reactivex/rxjs')
```

Comment:
I see, we should add it to `package.json` then. I thought at first that `@` makes it somehow work without installing :)